sdk/storage/azure_storage_queue/assets.json

```diff
@@ -1,6 +1,6 @@
 {
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "rust",
-  "Tag": "rust/azure_storage_queue_5f6f02d3be",
+  "Tag": "rust/azure_storage_queue_cda6f79f01",
```

Comment:
Why did you re-record? It would've been better that you didn't as a way of regression testing your TypeSpec changes. Or was something else wrong once you got past the incorrect status codes?

Reply:
> [Fix TypeSpec, re-record test for sanity](https://github.com/Azure/azure-sdk-for-rust/pull/2732/commits/0ba01091f52b0cb8c8101f3e4574a0507e5775fe)

It would be saner to _not_ re-record so that you're effectively regression testing against what the server returned, which shouldn't have changed.

Reply:
I can manually point it to the old recordings after I merge `main` in so we can use it as a regression test. I thumbed through the recordings and didn't see anything else to note- but usually just re-record for sanity when hitting all these CI issues and making code changes.